src/journal-remote/journal-remote.c

```diff
@@ -777,21 +789,43 @@ static int setup_microhttpd_socket(RemoteServer *s,
         return setup_microhttpd_server(s, fd, key, cert, trust);
 }
 
+static int null_timer_event_handler(sd_event_source *timer_event, uint64_t usec, void *userdata)
+{
+        return dispatch_http_event(timer_event, 0, 0, userdata);
+}
```

Comment:
please follow coding style, i.e. the opening bracket `{¸ should be on the same line as the function name. Also see CODING_STYLE.

Reply:
Sorry didn't see that was true for functions as well.